doc/vim-go.txt

```diff
@@ -1437,9 +1437,9 @@ Specifies the type of list to use for command outputs (such as errors from
 builds, results from static analysis commands, etc...). When an expected key
 is not present in the dictionary, |'g:go_list_type'| will be used instead.
 Supported keys are "GoBuild", "GoErrCheck", "GoFmt", "GoInstall", "GoLint",
-"GoMetaLinter", "GoModifyTags" (used for both :GoAddTags and :GoRemoveTags),
-"GoRename", "GoRun", and "GoTest".  Supported values for each command are
-"quickfix" and "locationlist".
+"GoMetaLinter", "GoMetaLinterAutoSave", "GoModifyTags" (used for both
+:GoAddTags and :GoRemoveTags), "GoRename", "GoRun", and "GoTest".  Supported
+values for each command are "quickfix" and "locationlist".
```

Comment:
Can you also update the doc for `g:go_metalinter_autosave` setting and `:GoMetaLinterAutoSaveToggle` command to add a notice that `locationlist` is used instead of `quickfix`. These two:

```
                                                  *'g:go_metalinter_autosave'*

Use this option to auto |:GoMetaLinter| on save. Only linter messages for
the active buffer will be shown. By default it's disabled >

  let g:go_metalinter_autosave = 0
<
```

and

```
                                                 *:GoMetaLinterAutoSaveToggle*
:GoMetaLinterAutoSaveToggle

    Toggles |'g:go_metalinter_autosave'|.
```

It's worth adding this because for example I use `quickfix` for everything and have custom mappings to select next/previous items from the list. So people should be aware of the side effects.

Reply:
done